repos/fdbt-site/src/pages/managePassengerTypes.tsx

```diff
@@ -294,17 +310,38 @@ const hasError = (errors: ErrorInfo[], name: string) => {
     return '';
 };
 
-export const getServerSideProps = (ctx: NextPageContextWithSession): { props: ManagePassengerTypesProps } => {
+export const getServerSideProps = async (
+    ctx: NextPageContextWithSession,
+): Promise<{ props: ManagePassengerTypesProps }> => {
+    let singlePassengerType: SinglePassengerType | undefined;
     const csrfToken = getCsrfToken(ctx);
 
-    const errorsFromSession = getSessionAttribute(ctx.req, MANAGE_PASSENGER_TYPE_ERRORS_ATTRIBUTE);
+    const passengerTypeId = Number(ctx.query.id);
+
+    const passengerTypeIdIsANumber = Number.isInteger(passengerTypeId);
+
+    const isInEditMode = passengerTypeIdIsANumber;
```

Comment:
Do you need the 2 variables or could you just have isInEditMode = Number.isInteger(passengerTypeId); ?